pkg/gardenlet/controller/backupentry/backup_entry_reconciler.go

```diff
@@ -81,6 +81,16 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	if be.DeletionTimestamp != nil {
 		return r.deleteBackupEntry(ctx, gardenClient, be)
 	}
+
+	if shouldMigrateBackupEntry(be) {
+		return r.migrateBackupEntry(ctx, gardenClient, be)
+	}
+
+	if !controllerutils.BackupEntryIsManagedByThisGardenlet(ctx, gardenClient.Client(), be, r.config) {
+		r.logger.Debugf("Skipping because BackupEntry is not managed by this gardenlet in seed %s", *be.Spec.SeedName)
+		return reconcile.Result{}, nil
+	}
```

Comment:
Shouldn't this check be earlier/before calling the actuator?

Reply:
I added this check just in case (exactly as we do it in the Shoot), because of a problem we encountered after the `Shoot` migration as the shoot could be requeued with the old `spec.SeedName` - https://github.com/gardener/gardener/pull/3301